service/src/main/resources/HTMLTemplate.html

```diff
@@ -165,8 +165,12 @@ <h1 class="heading heading--one">
         <div class="data-list__row data-list__row--no-border">
           <dt class="data-list__title">Applicant</dt>
           <dd class="data-list__definition">
-            {% if case_data.applicant is not empty and case_data.applicant.name is not empty %}
-              {{ case_data.applicant.name }}
+            {% if case_data.applicants is not empty %}
```

Comment:
Am I right saying that it will only fetch data from migrated cases? What about old ones?

Reply:
👍